app/(modals)/organizations/_features/organization-profile.schema.ts

```diff
@@ -10,14 +10,14 @@ const organizationSchema = z.object({
   instagram_link: z.string().url(),
   discord_link: z.string().url(),
   email: z.string().email(),
-  logo_url: z.string().url(),
+  logo_url: z.string().url().nullable(),
```

Comment:
Maybe .optional is what we want https://zod.dev/?id=optional but I'm not to familar with zod

Reply:
no .optional() is for undefined and .nullable() is for nullable content